Add editorial text on bandwidth probing techniques (#1477)
moq-wg/moq-transport: draft-ietf-moq-transport.md

app-limited for at least a full round trip to ensure the available bandwidth can be
measured.

Applications can address this by subscribing to additional tracks at a lower priority
or by using padding streams ({{padding-streams}}) to fill the congestion window

> during probing intervals without affecting the delivery of higher priority media.

Reviewer: ```suggestion
during probing intervals while minimizing the impact on higher priority media.
```
